pkg/ebpf/events_processor.go

```diff
@@ -313,6 +315,10 @@ func (t *Tracee) processEvent(event *trace.Event) error {
 		}
 
 	case events.HookedProcFops:
+		const (
```

Comment:
I would make it global but not sure where it would be appropriate (we have consts for filter maps but it's in the `filter.go` file which wouldn't make sense for these.
Because I didn't find a reasonable place for the consts I opted to make them locally scoped. I think golang uses consts like `#DEFINE` macros in C and inlines them so there shouldn't be a performance penalty.